.gitignore

```diff
@@ -54,3 +54,4 @@ release
 package-lock.json
 web-template-release/
 template-pages/sample/*
+/src/docs/examples/
```

Comment:
why ignore examples?

Reply:
these are examples copied from qld.gov.au just for debugging issues, exact pages so suppose some sensitive information is deleted on qld.gov.au but is available in SWE repo won't be good.